features/waarden.feature

```diff
@@ -6,83 +6,130 @@ Feature: leveren van waarden bij een WOZ-object
   Per waardepeildatum wordt één waarde bepaald.
   Voor een waardepeildatum wordt de waarde genomen met de meest recente ingangsdatum met betrekking tot die waardepeildatum.
 
-  Een waarde met alleen een beschikking status "vernietigingbeschikking" (02) en geen enkele andere status, wordt genegeerd.
-  
+  Een waarde met alleen een of meerdere beschikking status (beschikkingsStatussen) "vernietigingbeschikking" (02) en geen enkele andere status, wordt genegeerd.
+
   Waarden worden gesorteerd op waardepeildatum met de meest recente waardepeildatum eerst.
 
-  N.B. De API levert alleen actuele waarden. Dit betekent dat voor een waardepeildatum en ingangsdatum alleen de waarde wordt genomen met de meest recente beschikkingsdatum.
+  N.B. De API levert alleen actuele waarden. Dit betekent dat voor een waardepeildatum en ingangsdatum alleen de waarde wordt genomen met de meest recente beschikkingsdatum. Ook levert de API alleen de actuele beschikkingsStatussen.
   N.B. De provider van de API kan een maximum aantal op te nemen waarden bepalen.
   N.B. Niet elke waarde bij het WOZ-object hoeft te komen uit een beschikking op naam van de actuele belanghebbende(n).
 
   indicatieBezwaarBeroep krijgt de waarde true wanneer ten minste één beschikking status van de betreffende WOZ-waarde een van de volgende waarden heeft:
-    - bezwaar ingediend (10)
-    - beroep aangetekend (20)
-    - hoger beroep aangetekend (23)
-    - cassatie ingesteld (30)
+    - bezwaaringediend (10)
+    - beroepaangetekend (20)
+    - hogerberoepaangetekend (23)
+    - cassatieingesteld (30)
 
 
   Scenario: meerdere jaren met zelfde eigenaar en zonder bezwaar of beroep
```

Comment:
Is het relevant om aan te geven dat de eigenaar hetzelfde is? Dit kan nl. niet worden opgemaakt uit de response

Reply:
"met zelfde eigenaar' gaat over situatie in de werkelijkheid, heeft voor algoritme van de proxy geen gevolgen